### PR TITLE
{devel,vis}[foss/2018b] add missing gperf dep for Qt5 5.10.1 and new config for Qt5 5.11.2

### PR DESCRIPTION
**Pull request by jhein32**
configs QT5.10.1 (bugfix) and QT5.11.2 (new) for foss/2018b

This replaces PR #7082, which got corrupted when attempting a merge conflict resolution.

### CODE REVIEW

File: easybuild/easyconfigs/q/Qt5/Qt5-5.10.1-foss-2018b.eb [hunk 1]
```diff
@@ -24,6 +24,7 @@ dependencies = [
     ('libGLU', '9.0.0'),
     ('NSS', '3.39'),
     ('DBus', '1.13.6'),
+    ('gperf', '3.1'),
```
Comment (boegel) [inline easybuild/easyconfigs/q/Qt5/Qt5-5.10.1-foss-2018b.eb]: @jhein32 We should apply this same fix in the other easyconfigs for `Qt5` 5.10.1?

Comment (jhein32) [reply easybuild/easyconfigs/q/Qt5/Qt5-5.10.1-foss-2018b.eb]: I only had the issue in foss/2018b.  I could build intel/2018a and foss/2018a without issues. Most likely I have an older version that doesn't force a webengine.  gperf is only needed when the webengine is build - as noted in Slack, I had that failing and found a clear comment in the log: No gperf, no webengine.

Comment (jhein32) [reply easybuild/easyconfigs/q/Qt5/Qt5-5.10.1-foss-2018b.eb]: Ok, since you asked so nicely ;)  I put a revised Qt 5.10.1 for foss/2018a.

Comment (jhein32) [reply easybuild/easyconfigs/q/Qt5/Qt5-5.10.1-foss-2018b.eb]: I leave the intel/2018a config to be fixed once the webengine issues with intel have been fixed.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3103.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/dbca9f8cff6765ffde9c4f36e440992f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2606.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8679dd92b09ec3939b203cca4539a3ff for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node2002.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/abad0b1d60f42ddfedb156da2e08bb18 for a full test report.

File: easybuild/easyconfigs/q/Qt5/Qt5-5.10.1-foss-2018b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/q/Qt5/Qt5-5.10.1-foss-2018b.eb]: @jhein32 I added `gperf` as a dep in #7010

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3132.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/7f0071de6576abcd4d55ea00ff2bde56 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @jhein32!

**Pull request merged**